app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
 
     <string name="error_generic">An error occurred.</string>
+    <string name="error_network">Network error occurred</string>
```

Comment:
I think we can be more verbose and freindly here, there is a lot of space. How about something like "A network error occured. Please check your internet connection and try again!"